### PR TITLE
Lot size

### PR DESCRIPTION
**Pull request by kelvinlau20100**

### CODE REVIEW

**Comment by kelvinlau20100** [comment]
one thing that could be good is to make util functions for converting to native position size and back

**Comment by xjcaa** [comment]
> 

Can you add this

File: src/client.ts [hunk 1]
```diff
@@ -396,7 +397,7 @@ export class Client {
     let orderIx = await placeOrderIx(
       marketIndex,
       price,
-      size,
+      size * Math.pow(10, POSITION_PRECISION),
```
Comment (xjcaa) [inline src/client.ts:400]: Can we pass in the raw number - like 1000 = 1 lot

Comment (xjcaa) [reply src/client.ts:400]: And do the conversion on the outside

File: src/risk.ts [hunk 2]
```diff
@@ -229,7 +247,9 @@ export function calculateLiquidationPrice(
     return 0;
   }
   let availableBalance = accountBalance - marginRequirement + unrealizedPnl;
-  return markPrice - availableBalance / position;
+  return (
+    markPrice - availableBalance / (position / Math.pow(10, POSITION_PRECISION))
```
Comment (xjcaa) [inline src/risk.ts:251]: Can we just make 1_000 a constant.

File: src/client.ts [hunk 3]
```diff
@@ -610,7 +611,7 @@ export class Client {
       this._marginAccountAddress,
       market,
       liquidatedMarginAccount,
-      size
+      size * Math.pow(10, POSITION_PRECISION)
```
Comment (xjcaa) [inline src/client.ts:614]: This needs to be fixed

**Pull request merged**